doc/classes/Resource.xml

```diff
@@ -104,29 +104,29 @@
 		<method name="is_built_in" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the resource is built-in (from the engine) or [code]false[/code] if it is user-defined.
+				Returns [code]true[/code] if the resource is saved on disk as a part of another resource's file.
 			</description>
 		</method>
 		<method name="reset_state">
 			<return type="void" />
 			<description>
-				For resources that use a variable number of properties, either via [method Object._validate_property] or [method Object._get_property_list], override [method _reset_state] to correctly clear the resource's state.
+				Makes the resource clear its non-exported properties. Useful when implementing a custom resource format by extending [ResourceFormatLoader] and [ResourceFormatSaver].
```

Comment:
It may be good to add somethiing like, "See also [method _reset_state]".

I still don't quite get why we need both. @dsnopek, couldn't just `reset_state` be overridable in GDExtension?